packages/react-charting/src/utilities/gradients.ts

```diff
@@ -0,0 +1,146 @@
+export const DataVizGradientPalette = {
+  gradient1: 'default.1',
+  gradient2: 'default.2',
+  gradient3: 'default.3',
+  gradient4: 'default.4',
+  gradient5: 'default.5',
+  gradient6: 'default.6',
+  gradient7: 'default.7',
+  gradient8: 'default.8',
+  gradient9: 'default.9',
+  gradient10: 'default.10',
+  // extended gradients
+  /* gradient1Ext: 'extended.1',
+  gradient2Ext: 'extended.2',
+  gradient3Ext: 'extended.3',
+  gradient4Ext: 'extended.4',
+  gradient5Ext: 'extended.5',
+  gradient6Ext: 'extended.6',
+  gradient7Ext: 'extended.7',
+  gradient8Ext: 'extended.8',
+  gradient9Ext: 'extended.9',
+  gradient10Ext: 'extended.10', */
+  // status gradients
+  success: 'semantic.success',
+  highSuccess: 'semantic.highSuccess',
+  warning: 'semantic.warning',
+  error: 'semantic.error',
+  highError: 'semantic.highError',
+  disabled: 'semantic.disabled',
+};
+
+/**
+ * Key: Color code.
+ * Value:
+ * Index 0 - Default gradient for light theme,
```

Comment:
mention that 0th index in value represents startColor and 1st index represents endColor